ExtraDry/ExtraDry.Core/Core/InputFormatAttribute.cs

```diff
@@ -13,12 +13,14 @@ public class InputFormatAttribute : Attribute
     public Type? DataTypeOverride { get; set; }
 
     /// <summary>
-    /// The icon to be used in the display of this propertys input, typically on the left of the input
+    /// The icon to be used in the display of this property's input, typically on the left of the input
     /// </summary>
     public string? Icon { get; set; }
 
     /// <summary>
-    /// The affordance icon to be used in the display of this propertys input, typically on the right of the input.
+    /// The affordance icon to be used in the display of this property's input, typically on the right of the input.
     /// </summary>
     public string? Affordance { get; set; }
+
+    public PropertySize Size { get; set; } = PropertySize.Calculated;

```

Comment:
Having the properties on attributes nullable is used by Microsoft attribute (e.g. Display) and this attribute to allow code to determine if the attribute has a value that has been set or not.  There is another more robust pattern for doing this which has multiple backing fields and overrides set/get to not allow users to explicitly set the value to null and circumvent this.  Anyway, going with the simpler version of null detection, should probably keep it consistent and make this a `PropertySize?`.

Reply:
I've followed the Microsoft pattern for this property as using `PropertySize?` gave a compiler error.